pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/ElasticSearchUtil.java

```diff
@@ -196,7 +196,7 @@ public static RestHighLevelClient getClientForElasticsearchCloud(@NonNull String
                         httpClientBuilder.setDefaultIOReactorConfig(IOReactorConfig.custom().setSoKeepAlive(true).build());
                     }
                     return httpClientBuilder;
-                });
+                }).setMaxRetryTimeoutMillis(100000);
```

Comment:
not sure if this one was added recently or is part of revert since it seems nothing was there previously

Reply:
oh - saw the other PR - the above method was previously taken out and is being deprecated - so yep, reverted